packages/shared/src/enterprise/decision-criteria/statusIndicatorData.ts

```diff
@@ -0,0 +1,187 @@
+import {
```

Comment:
While some of these statuses in this file are enterprise, I don't think all of them are.

Thoughts about moving this outside of the enterprise folder? We can keep the functions that create the `ExperimentResultStatusData` in enterprise so we can differentiate between what's paid and what's not.